lib/cli.js

```diff
@@ -157,6 +157,21 @@ function runDevelopmentMode (testModulePath) {
     callback()
   })
 
+  vorpal.command('runto <stepNumber>')
+  .description('Run from current step to step number')
+  .action(function (args, callback) {
+    handleRun(co(function* () {
```

Comment:
It should also check if target step exists.

If not, it should refuse to run.

Another edge case here is that if the action step is 5.1, user should be able to say `runto 5` and it should pause at 5.1.